src/pages/home/report/ReportActionsList.js

```diff
@@ -385,13 +410,16 @@ function ReportActionsList({
                     style={styles.overscrollBehaviorContain}
                     data={sortedReportActions}
                     renderItem={renderItem}
-                    contentContainerStyle={styles.chatContentScrollView}
+                    contentContainerStyle={contentContainerStyle}
                     keyExtractor={keyExtractor}
                     initialRowHeight={32}
                     initialNumToRender={initialNumToRender}
-                    onEndReached={loadMoreChats}
+                    onEndReached={loadOlderChats}
                     onEndReachedThreshold={0.75}
-                    ListFooterComponent={renderFooter}
+                    onStartReached={loadNewerChats}
+                    onStartReachedThreshold={0.75}
```

Comment:
May I ask why this is needed?
I mean why we need to load newer actions? When we open report, I think we get all the latest actions.
@perunt Will you explain?